OpenRA.Mods.Common/Traits/SpawnActorOnDeath.cs

```diff
@@ -90,6 +97,10 @@ void INotifyRemovedFromWorld.RemovedFromWorld(Actor self)
 			if (attackingPlayer == null)
 				return;
 
+			var defeated = self.Owner.WinState != WinState.Undefined;
```

Comment:
This should be checking `== WinState.Defeated`.  I know this is pedantic because the game stops when somebody wins, but the name should still match the implementation.